lib/index.js

```diff
@@ -353,6 +354,71 @@ const feedbackHandler = async function feedbackHandler(options) {
   logger.info('Thank you!');
 };
 
+async function triggerUpsertHandler(options) {
+  // We have to use the position number here because of lack of good support
+  // for nested commands with positional arguments
+  // consider replacing with `nested-yargs` npm package
+  const triggerName = options._[2];
+  const triggerOptions = {
+    source: {
+      uri: options.sourceUri,
+    },
+    target: {
+      funcName: options.targetFunc,
+    },
+  };
+  if (options.errorHandlerFunc) {
+    triggerOptions.errorHandler = {
+      funcName: options.errorHandlerFunc,
+    };
+  }
+  const result = await triggers.upsert(triggerName, triggerOptions);
+  logger.info(options.json ? JSON.stringify(result) : result.status);
```

Comment:
why stringifying the entire result but printing only status?

Reply:
Removing `json` option